### DISCUSSION TITLE
i need help

### DISCUSSION THREAD

**Discussion by dylzohtml555** [mkleehammer/pyodbc#1244] [Q&A]
i keep getting a error 
import pypyodbc

# Replace 'C:\Users\Melmc\OneDrive - Glow Scotland\Dylz1.accdb' with the actual full path to your Dylz1.accdb file
db_file = r'C:\Users\Melmc\OneDrive - Glow Scotland\Dylz1.accdb'
conn_str = f'Driver={{Microsoft Access Driver (*.mdb, *.accdb)}};DBQ={db_file};'

conn = None  # Initialize conn outside the try block

try:
    conn = pypyodbc.connect(conn_str)
    cursor = conn.cursor()

    # Create a table with an auto-incrementing ID field
    cursor.execute('''
        CREATE TABLE SampleTable (
            ID INT AUTOINCREMENT PRIMARY KEY,
            Name TEXT NOT NULL,
            Age INT
        )
    ''')
    conn.commit()

    # Insert data into the table
    cursor.execute("INSERT INTO SampleTable (Name, Age) VALUES (?, ?)", ("John Doe", 30))
    conn.commit()

    # Fetch data from the table
    cursor.execute("SELECT * FROM SampleTable")
    rows = cursor.fetchall()
    for row in rows:
        print(row)

except Exception as e:
    print("An error occurred:", e)

finally:
    # Make sure to close the connection after use
    if conn is not None:
        conn.close()

**Comment by gordthompson** [comment]
> import pypyodbc

You are using [pypyodbc](https://pypi.org/project/pypyodbc/), not pyodbc. They are not the same thing.

Also, you don't even mention what the error is, or where it occurs.

Try asking for help on Stack Overflow

https://stackoverflow.com/questions/tagged/pypyodbc

**Reply by dylzohtml555** [reply]
Hi there is a error if u look
